safe-global/web-core
# Feature: Added Safe Apps section in the Dashboard

## What it solves

Resolves #581 

## How this PR fixes it

Added a new Component called `<SafeAppsDashboardSection>` in the Dashboard

Added a new hook called `useRankedSafeApps`

## How to test it

Just open the Dashboard and check the Safe Apps Section.

Most used Apps should appear in the first positions

## Screenshots

![Captura de pantalla 2022-09-22 a las 11 21 48](https://user-images.githubusercontent.com/26763673/191709787-06900ce9-725e-4edc-8222-218774d68324.png)
